net/netifyd/Makefile

```diff
@@ -0,0 +1,113 @@
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+# Name, version and release number
+# The name and version of your package are used to define the variable to point to the build directory of your package: $(PKG_BUILD_DIR)
+PKG_NAME:=netifyd
+PKG_VERSION:=2.67
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Darryl Sokoloski <darryl@egloo.ca>
+PKG_LICENSE:=GPL-3.0+
+
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS:=CONFIG_LIBCURL_ZLIB
+
+# Source settings
+
+# Disable the following and enable SOURCE_DIR below for local test builds.
```

Comment:
keep all this sort of stuff in your own feed if you like, it has no place in the main feed.  (And might I direct your attention to https://openwrt.org/docs/guide-developer/packages#working_on_local_application_source ? (I used to do things like this in my own private feeds too, but package source override is easier)